tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -9111,6 +9110,440 @@ TEST_P(OpConverter_FP32_FP16_Test, ConvertPad) {
+    const auto rank_out =
+        std::accumulate(std::begin(expect_dims), std::end(expect_dims), 1,
+                        std::multiplies<int>());
+
+    assert(rank_out == (expected_out ? expected_out->size() : rank[0]));
+    expected_output.resize(rank_out);
+    const auto& data_then = *par_value[1];
+    const auto& data_else = *par_value[2];
+    for (int i = 0; i < rank_out; i++) {
+      expected_output[i] = expected_out
+                               ? (*expected_out)[i]
+                               : data_cond.at(i) ? data_then[i] : data_else[i];
+    }
+  };
+
+  auto run_test = [&](const NodeDef& node, const std::vector<int>& exp_dims) {
+    for (int n = 0; n < 2; n++) {
+      set_parameters();
+      TestOpConverter("my_select", node, exp_dims, Status::OK(), Status::OK(),
+                      ElementsAreArray(expected_output));
+      if (!n) {
+        // Changing the condition and expected_output.
+        for (auto idx = data_cond.size(); idx--;)
+          data_cond[idx] = 1 - data_cond[idx];
+
+        // Compare of the shapes if the tensors "then" and "else".
+        if (*par_dims[1] != *par_dims[2]) {
+          // Shapes are different:
+          //     assigning +1's and -1's to the elements
+          //     of the tensors "then" and "else", respectively
+          for (int p = 1; p <= 2; p++) {
+            auto& values = *par_value[p];
+            const auto val = p == 1 ? 1 : -1;
+            for (auto idx = values.size(); idx--;) values[idx] = val;
+          }
+          //    and set the appropriate expected values.
+          for (auto idx = expected_output.size(); idx--;)
+            expected_output[idx] = expected_output[idx] > 0 ? -1 : 1;
+        } else {
+          // Shapes are the same:
+          //    just change the signs of the expected values.
+          for (auto idx = expected_output.size(); idx--;)
+            expected_output[idx] = -expected_output[idx];
+        }
+      }
+    }
+  };
+
+  std::array<DataType, 3> data_types = {DT_FLOAT, DT_HALF, DT_INT32};
+  Scope s = Scope::NewRootScope();
+  const auto op =
+      ops::SelectV2(s.WithOpName("my_select"),
+                    ops::Placeholder(s.WithOpName(par_name[0]), par_type[0]),
+                    ops::Placeholder(s.WithOpName(par_name[1]), par_type[1]),
+                    ops::Placeholder(s.WithOpName(par_name[2]), par_type[2]));
+  const auto& node = op.operation.node()->def();
+
+  std::vector<std::vector<int>> dims_params = {{8}, {8, 2, 4}, {32, 32, 3200}};
+
+  // All parameters passed as the weights OR 1-element tensors.
+  par_dims = {&dims_params[0], &dims_params[0], &dims_params[0]};
+  if (trt_mode_ == TrtTestMode::kImplicitBatch) {
+    do {
+      set_parameters();
+      RunValidationAndConversion(
+          node, error::UNIMPLEMENTED,
+          "Conversion for SelectV2 is not supported in implicit batch mode.");
+    } while (nextTensorWeigtConfiguration(config));
```

Comment:
error: use of undeclared identifier 'nextTensorWeigtConfiguration'

Reply:
@bixia1 : I am not sure why `nextTensorWeigtConfiguration` is undeclared. This function is defined [in these lines](https://github.com/tensorflow/tensorflow/blob/690d347af047876756c492024dd502f7497826ee/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc#L3953-L3958) of `convert_nodes_test.cc`. Similar function was implemented in one of the previous PRs, but it was defined as a lambda function. So I think it might be some sort of merge issue.
Anyway, I'll fix the typo and change the name of this function from  `nextTensorWeigtConfiguration` to  `nextTensorWeightConfiguration`.

Reply:
The function definition is protected via #if IS_TRT_VERSION_GE(8, 2, 0, 0)
but the use of it is not.

Reply:
@bixia1 : I'm sorry I didn't notice it. Corrected.